index.md

```diff
@@ -60,50 +62,60 @@ First, we need to create a "repository". This is the place where your backups wi
 
 In order to create a repository at `/tmp/backup`, run the following command and enter the same password twice:
 
-    $ restic init --repository /tmp/backup init
-    enter password for new backend:
-    enter password again:
-    created restic backend 085b3c76b9 at /tmp/backup
-    Please note that knowledge of your password is required to access the repository.
-    Losing your password means that your data is irrecoverably lost.
+{% highlight console %}
+$ restic init --repository /tmp/backup init
+ enter password for new backend:
```

Comment:
the indentation changed (there is a space now that wasn't there before). intentional?